admin/hotlist-service/src/main/java/io/mosip/hotlist/config/SwaggerConfig.java

```diff
@@ -49,11 +48,11 @@ public OpenAPI openApi() {
 		return api;
     }
 	
-	@Bean
+	/*@Bean
```

Comment:
Why is this commented? Can you check by uncommenting?

Reply:
Uncommented  and imported the valid imports for GroupedOpenApi, working as expected